doc/man3/SSL_CTX_set_stateless_cookie_generate_cb.pod

```diff
@@ -42,9 +42,15 @@ verify_stateless_cookie_cb() is called.
 
 Neither function returns a value.
 
+=head1 HISTORY
+
+Legacy setter methods SSL_CTX_set_cookie_generate_cb() and SSL_CTX_set_cookie_verify_cb()
+are still available for current applications who want support both TLS1.2 with stateful
+cookies and TLS1.3 with stateless cookies.
```

Comment:
This paragraph is not entirely correct. SSL_CTX_set_cookie_generate_cb() and SSL_CTX_set_cookie_verify_cb() are for use in *DTLS* applications (not TLS1.2). Since there is no DTLSv1.3 these are not legacy. If you want to do cookies in DTLS you have to use the older functions. See the DTLSv1_listen() docs for some discussion of that.

Reply:
Edid

Reply:
Thank you. I had been confused since the manual page for `SSL_CTX_set_cookie_generate_cb()` was missing.